src/utils/discordClient.js

```diff
@@ -0,0 +1,18 @@
+
+require("dotenv").config();
+const {
+  Client,
+  IntentsBitField,
+  GatewayIntentBits,
+} = require("discord.js");
+
+const client = new Client({
```

Comment:
Make this into a class like `DiscordClient` which handles `login` (`client.login()`) into the Discord Gateway (i.e API), and registers the `interactionCreate`, `message`, `ready` handlers in its own methods (e.g `HandleReadyEvent` to handle the `on('ready')`).

Then use this class in the `index.js` file to run an abstracted Discord bot instance.